testing/regress/ecl/despray.ecl

```diff
@@ -0,0 +1,223 @@
+p2 := PROJECT(NOFOLD(dst2), t(LEFT));
+c2 := CATCH(p2, ONFAIL(TRANSFORM(rec,
+                                 SELF.result := 'Fail',
+                                 SELF.destFile := DestFile1,
+                                 SELF.sourceFile := SourceFile,
+                                 SELF.ip := '',
+                                 SELF.allowOverwrite := True,
+                                 SELF.msg := FAILMESSAGE
+                                )));
+#if (VERBOSE = 1)
+    output(c2);
+#else
+    output(c2, {result});
+#end
+
+DestFile3 := '/var/lib/HPCCSystems/mydropzone/../' + File;
+dst3 := NOFOLD(DATASET([{SourceFile, DestFile3, SrcAddrIp, True, '', ''}], rec));
+p3 := PROJECT(NOFOLD(dst3), t(LEFT));
+c3 := CATCH(p3, ONFAIL(TRANSFORM(rec,
+                                 SELF.result := 'Fail',
+                                 SELF.destFile := DestFile1,
+                                 SELF.sourceFile := SourceFile,
+                                 SELF.ip := '',
+                                 SELF.allowOverwrite := True,
+                                 SELF.msg := FAILMESSAGE
+                                )));
+#if (VERBOSE = 1)
+    output(c3);
+#else
+    output(c3, {result});
+#end
+
+DestFile4 := '/var/lib/HPCCSystems/mydropzona/' + File;
+dst4 := NOFOLD(DATASET([{SourceFile, DestFile4, SrcAddrIp, True, '', ''}], rec));
+p4 := PROJECT(NOFOLD(dst4), t(LEFT));
+c4 := CATCH(p4, ONFAIL(TRANSFORM(rec,
+                                 SELF.result := 'Fail',
+                                 SELF.destFile := DestFile1,
+                                 SELF.sourceFile := SourceFile,
+                                 SELF.ip := '',
+                                 SELF.allowOverwrite := True,
+                                 SELF.msg := FAILMESSAGE
+                                )));
+#if (VERBOSE = 1)
+    output(c4);
+#else
+    output(c4, {result});
+#end
+
+DestFile5 := '/var/lib/HPCCSystems/' + File;
+dst5 := NOFOLD(DATASET([{SourceFile, DestFile5, SrcAddrIp, True, '', ''}], rec));
+p5 := PROJECT(NOFOLD(dst5), t(LEFT));
+c5 := CATCH(p5, ONFAIL(TRANSFORM(rec,
+                                 SELF.result := 'Fail',
+                                 SELF.destFile := DestFile1,
+                                 SELF.sourceFile := SourceFile,
+                                 SELF.ip := '',
+                                 SELF.allowOverwrite := True,
+                                 SELF.msg := FAILMESSAGE
+                                )));
+#if (VERBOSE = 1)
+    output(c5);
+#else
+    output(c5, {result});
+#end
+
+DestFile6 := '/var/lib/HPCCSystems/mydropzone../' + File;
+dst6 := NOFOLD(DATASET([{SourceFile, DestFile6, SrcAddrIp, True, '', ''}], rec));
+p6 := PROJECT(NOFOLD(dst6), t(LEFT));
+c6 := CATCH(p6, ONFAIL(TRANSFORM(rec,
+                                 SELF.result := 'Fail',
+                                 SELF.destFile := DestFile1,
```

Comment:
Should be testFile6 (and see other examples).  The transform could be commoned up between examples, but may not be worth it.